lib/src/view/broadcast/broadcast_players_tab.dart

```diff
@@ -118,7 +121,7 @@ class _PlayersListState extends ConsumerState<PlayersList> {
   @override
   Widget build(BuildContext context) {
     final double eloWidth = max(MediaQuery.sizeOf(context).width * 0.2, 100);
-    final double scoreWidth = max(MediaQuery.sizeOf(context).width * 0.15, 70);
+    final double scoreWidth = max(MediaQuery.sizeOf(context).width * 0.15, 90);
```

Comment:
Why this change? Have you noticed a case where it was too small?

Reply:
This is because I changed the score from 'x/y' to 'x / y'.